include/cppflow/model.h

```diff
@@ -143,15 +143,15 @@ namespace cppflow {
 
     inline std::tuple<std::string, int> parse_name(const std::string& name) {
         auto idx = name.find(':');
-        return (idx == -1 ? std::make_tuple(name, 0) : std::make_tuple(name.substr(0, idx), std::stoi(name.substr(idx + 1))));
+        return (idx == std::string::npos ? std::make_tuple(name, 0) : std::make_tuple(name.substr(0, idx), std::stoi(name.substr(idx + 1))));
     }
 
     inline std::vector<tensor> model::operator()(std::vector<std::tuple<std::string, tensor>> inputs, std::vector<std::string> outputs) {
 
         std::vector<TF_Output> inp_ops(inputs.size());
         std::vector<TF_Tensor*> inp_val(inputs.size(), nullptr);
 
-        for (int i=0; i<inputs.size(); i++) {
+        for (decltype(inputs.size()) i=0; i<inputs.size(); i++) {
```

Comment:
I went back and forth on whether to use something more readable like `auto i=0u` in place of `decltype`. I felt that this particular call should be made as portable as possible, making the code a bit less readable (at least to me). If you disagree with the approach, then I can switch to a more readable approach.

There is a brief discussion on this topic over at https://stackoverflow.com/questions/17505164/using-auto-in-loops-c if you want a bit more information.